### PR TITLE
adding assessment_stage, chance_level, and item_id to corpora

### PR DESCRIPTION
**Pull request by kachergis**
no longer using `notes` to indicate practice trials; now `assessment_stage = {practice_response, test_response, instructions}`

### CODE REVIEW

**Comment by kachergis** [comment]
I'm realizing that SDS scoring needs to be updated: we now have a new background color feature (white is default; other options are black/striped/gray), and not all blocks vary all dimensions: so whenever a dimension has no variability (e.g., all items are same size), we need to ignore that dimension for scoring.

**Review by Zio-4 — COMMENTED** [review]
assessment_stage isn't writing to db for AFC tasks

**Comment by kachergis** [comment]
@Zio-4 , did Zach's new commit fix what you commented on? (we want to save assessmentStage / assessment_stage to DB for AFC tasks -- not sure if you meant we were doing it improperly, or if a change elsewhere needs to be made in order for it to save?)

can you re-review the latest? I finished re-doing SDS scoring and according to my tests it works, but would love to have your eyes on it.

File: task-launcher/src/tasks/mental-rotation/trials/instructions.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ export const imageInstructions = {
     const t = taskStore().translations;
     return `<button class="primary">${t.continueButtonText}</button>`;
   },
-  keyboard_choices: 'ALL_KEYS',
+  keyboard_choices: 'NO_KEYS',
```
[the remark this answers is not in the document]
Comment (kachergis) [reply task-launcher/src/tasks/mental-rotation/trials/instructions.js:115]: it's too easy to mistakenly go through IMO..should probably just put a delay to go through instructions (until audio is done)

**Pull request merged**